experiments/common_pile/tokenize_common_pile.py

```diff
@@ -0,0 +1,503 @@
+        hf_dataset_id="common-pile/uspto_filtered",
+        revision="13894c5",
+        gcs_output_path=this_output_path(),
+        wait_for_completion=True,
+    ),
+    override_output_path="raw/common_pile/uspto_filtered-13894c5",
+)
+
+wikimedia_filtered = ExecutorStep(
+    name="raw/common_pile/wikimedia_filtered",
+    fn=download_hf,
+    config=DownloadConfig(
+        hf_dataset_id="common-pile/wikimedia_filtered",
+        revision="0641bb8",
+        gcs_output_path=this_output_path(),
+        wait_for_completion=True,
+    ),
+    override_output_path="raw/common_pile/wikimedia_filtered-0641bb8",
+)
+
+wikiteam_filtered = ExecutorStep(
+    name="raw/common_pile/wikiteam_filtered",
+    fn=download_hf,
+    config=DownloadConfig(
+        hf_dataset_id="common-pile/wikiteam_filtered",
+        revision="f4ed055",
+        gcs_output_path=this_output_path(),
+        wait_for_completion=True,
+    ),
+    override_output_path="raw/common_pile/wikiteam_filtered-f4ed055",
+)
+
+youtube_filtered = ExecutorStep(
+    name="raw/common_pile/youtube_filtered",
+    fn=download_hf,
+    config=DownloadConfig(
+        hf_dataset_id="common-pile/youtube_filtered",
+        revision="dff8c8a",
+        gcs_output_path=this_output_path(),
+        wait_for_completion=True,
+    ),
+    override_output_path="raw/common_pile/youtube_filtered-dff8c8a",
+)
+
+# Map dataset names to their corresponding raw download steps
+COMMON_PILE_DATASETS: dict[str, TokenizerStep] = {
+    "arxiv_abstracts": arxiv_abstracts_filtered,
+    "arxiv_papers": arxiv_papers_filtered,
+    "biodiversity_heritage_library": biodiversity_heritage_library_filtered,
+    "caselaw_access_project": caselaw_access_project_filtered,
+    "cccc": cccc_filtered,
+    "data_provenance_initiative": data_provenance_initiative_filtered,
+    "doab": doab_filtered,
+    "foodista": foodista_filtered,
+    "github_archive": github_archive_filtered,
+    "library_of_congress": library_of_congress_filtered,
+    "libretexts": libretexts_filtered,
+    "news": news_filtered,
+    "oercommons": oercommons_filtered,
+    "peS2o": peS2o_filtered,
+    "pre_1929_books": pre_1929_books_filtered,
+    "pressbooks": pressbooks_filtered,
+    "project_gutenberg": project_gutenberg_filtered,
+    "public_domain_review": public_domain_review_filtered,
+    "pubmed": pubmed_filtered,
+    "python_enhancement_proposals": python_enhancement_proposals_filtered,
+    "regulations": regulations_filtered,
+    "stackexchange": stackexchange_filtered,
+    "stackv2_edu": stackv2_edu_filtered,
+    "stackv2_html": stackv2_html_filtered,
+    "ubuntu_irc": ubuntu_irc_filtered,
+    "uk_hansard": uk_hansard_filtered,
+    "usgpo": usgpo_filtered,
+    "uspto": uspto_filtered,
+    "wikimedia": wikimedia_filtered,
+    "wikiteam": wikiteam_filtered,
+    "youtube": youtube_filtered,
+}
+
+# Effective token counts for the main training stage (in teratokens)
+COMMA_MAIN_MIXTURE_WEIGHTS = {
```

Comment:
why "comma"?

Reply:
The data mixture weights are specific to https://huggingface.co/datasets/common-pile/comma_v0.1_training_dataset/tree/main

These are the weights they used to train their "Comma" language model. Seems worth separating out the weights used for a particular model from the underlying data.